easybuild/easyconfigs/m/magma/magma-2.5.1-CrayGNU-19.10-cuda-10.1.eb

```diff
@@ -28,11 +27,13 @@ files_to_copy = ["lib", "include", "testing", "docs", "README", (['control/*.h']
 
 modextravars = { 'LD_LIBRARY_PATH':'$::env(CRAY_LD_LIBRARY_PATH):$::env(LD_LIBRARY_PATH)'}
 
-prebuildopts = 'mv make.inc-gcc-2.5.1 make.inc && CUDADIR=$EBROOTCUDA '
+prebuildopts = 'module unload cray-libsci && module list && mv make.inc-gcc-2.5.1 make.inc && CUDADIR=$CRAY_CUDATOOLKIT_DIR '
```

Comment:
If you pin the cudatoolkit version, you can still rely on the `$EBROOTCUDA` variable (we've removed the versionless cudatoolkit entry from the metadata file, that's probably why you saw this variable empty, see [here](https://github.com/eth-cscs/production/blob/master/easybuild/cray_external_modules_metadata-19.10.cfg#L81))